f5lbaasdriver/v2/bigip/device_scheduler.py

```diff
@@ -287,23 +292,34 @@ class CapacityFilter(DeviceFilter):
 
     def __init__(self, scheduler):
         super(CapacityFilter, self).__init__(scheduler)
-        self.load_constant()
+        self.load_config()
 
-    def load_constant(self):
-        super(CapacityFilter, self).load_constant()
+    def load_config(self):
+        super(CapacityFilter, self).load_config()
 
-        if "flavor" not in self.constant:
-            self.constant["flavor"] = constants_v2.FLAVOR_CONN_MAP
+        if self.cust_cfg.get("flavor"):
+            LOG.debug("merge config: customized flavors")
+            config.merge_cfg_dict(
+                self.constant["flavor"],
+                self.cust_cfg["flavor"]
+            )
 
-        if "capacity" not in self.constant:
-            self.constant["capacity"] = constants_v2.CAPACITY_MAP
+        if self.cust_cfg.get("capacity") and \
+                self.cust_cfg.get("capacity").get("license"):
+            LOG.debug(
+                "merge config: customized capacity licenses"
+            )
+            config.merge_cfg_dict(
+                self.constant["capacity"]["license"],
+                self.cust_cfg["capacity"]["license"]
+            )
 
         capacity_const = self.constant["capacity"]
         if "license" in capacity_const:
             for lic in capacity_const["license"]:
                 lic_const = capacity_const["license"][lic]
                 # Negative value means unlimited
-                for key in ["bandwidth", "lod", "rod", "cod"]:
```

Comment:
We cannot remove "bandwidth", because the inherited BandwidthCapacityFilter requires it.